src/components/Inputs/Dropdown/styles.tsx

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components/native';
+import { Colors } from 'assets/Colors';
+
+export const ButtonContainer = styled.View`
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+export const PlatformContainer = styled.View`
```

Comment:
add margin here to match/standardize spacing between description + box of other questions 
<img width="338" alt="Screen Shot 2021-12-05 at 12 24 16 AM" src="https://user-images.githubusercontent.com/63089231/144739234-378632bf-cba0-4d1d-9652-546a98a19707.png">